### PR TITLE
Removing the floating navigation bar, aka Stan

### LINKED ISSUE #556: Detail Page - Remove sidebar nav from layout

### PR DESCRIPTION
**Pull request by ShimiSun**
This is the best PR I filed this month, removing this hideous component is fulfilling from both design & dev points.

Cheers for cleaner code 🥂 

Fixes #556

### CODE REVIEW

File: aries-site/src/layouts/main/Layout.js [hunk 1]
```diff
@@ -12,13 +12,7 @@ const calcPad = size => {
...
 };
 
-export const Layout = ({
-  children,
-  descriptiveHeader,
-  title,
-  isLanding,
-  isNavPage,
-}) => {
+export const Layout = ({ children, descriptiveHeader, title, isLanding }) => {
```
Comment (taysea) [inline aries-site/src/layouts/main/Layout.js]: I think we can get rid of isLanding as well -- now that everything is full width. Then in `pages/index.js` we can remove the `isLanding` prop from the Layout tag.

File: aries-site/src/layouts/main/Layout.js [hunk 2]
```diff
@@ -12,13 +12,7 @@ const calcPad = size => {
   return val;
 };
 
-export const Layout = ({
-  children,
-  descriptiveHeader,
-  title,
-  isLanding,
-  isNavPage,
```
Comment (taysea) [inline aries-site/src/layouts/main/Layout.js]: We can remove the `isNavPage` prop from NavPage of `Guidelines`, `Foundation`, `Templates`, `Components`, `Design`, `Develop`, and `Resources`

File: aries-site/src/layouts/main/Layout.js [hunk 3]
```diff
@@ -96,13 +85,11 @@ Layout.propTypes = {
   children: PropTypes.oneOfType([PropTypes.element, PropTypes.array]),
   descriptiveHeader: PropTypes.element,
   isLanding: PropTypes.bool,
```
Comment (taysea) [inline aries-site/src/layouts/main/Layout.js]: Remove this if we agree that we no longer need `isLanding` at all.

File: aries-site/src/layouts/main/Layout.js [hunk 4]
```diff
@@ -96,13 +85,11 @@ Layout.propTypes = {
...
-  isNavPage: PropTypes.bool,
   title: PropTypes.string,
 };
 
 Layout.defaultProps = {
   descriptiveHeader: undefined,
   title: undefined,
   isLanding: false,
```
Comment (taysea) [inline aries-site/src/layouts/main/Layout.js]: Same as above.

File: aries-site/src/layouts/main/Layout.js (hunk 1 above)
Comment (ShimiSun) [reply aries-site/src/layouts/main/Layout.js]: I wish, seems like isLanding is needed for Footer rendering condition.

File: aries-site/src/layouts/main/Layout.js (hunk 2 above)
Comment (ShimiSun) [reply aries-site/src/layouts/main/Layout.js]: Music to my ears!

File: aries-site/src/layouts/main/Layout.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/layouts/main/Layout.js]: You're right, good call.

**Pull request merged**